### PR TITLE
Fix file import/export doc

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

**Comment by pentacular** [comment]
There's some leakage from an incomplete rebuild -- I'll resolve that once the other PR is in.

But if you have a chance, ptal at the .md and .nb diffs.

File: nb/documentation/file_import_and_export.nb [hunk 1]
```diff
@@ -2,19 +2,25 @@ md`
 # File IO
 JSxCAD supports importing and exporting .svg and .stl file types`;
 
+md`
+One thing to note is that output may consist of multiple 'pages'.
+
+A filename like 'teapot' will have the page number and filetype appended, so it may produce 'teapot_0.stl', 'teapot_1.stl', etc.
```
Comment (BarbourSmith) [inline nb/documentation/file_import_and_export.nb:8]: Good addition, I wasn't clear on how that worked

File: nb/documentation/file_import_and_export.nb [hunk 2]
```diff
@@ -2,19 +2,25 @@ md`
...
+`;
+
 md`
 ---
 ### Exporting .stl
 Creates an option to download the shape as an stl file. A view is created to show what will be downloaded.`;
 
-Arc(10, 10, 10).stl('fileName.stl');
```
Comment (BarbourSmith) [inline nb/documentation/file_import_and_export.nb:10]: Thanks :)

**Review by BarbourSmith — APPROVED** [review]
The changes and additions you made are excellent 👍 👍

**Pull request merged**